File: lab6/src/main/java/Boat.java
public class Boat extends Equipment {

    private int peopleCount;
    private int vesla;
    private int chairs;
    private int vest;

    public Boat(String ProduceName, double weight, int peopleCount,SportType sportType, int vesla, int chairs, int vest) {
        super(ProduceName, weight, SportType.KIAYKI);
        this.peopleCount = peopleCount;
        this.vesla = vesla;
        this.chairs = chairs;
        this.vest = vest;

    }

    public int getPeopleCount() {
        return peopleCount;
    }

    public void setPeopleCount(int peopleCount) {
        this.peopleCount = peopleCount;
    }

    public int getVesla() {
        return vesla;
    }

    public void setVesla(int vesla) {
        this.vesla = vesla;
    }

    public int getChairs() {
        return chairs;
    }

    public void setChairs(int chairs) {
        this.chairs = chairs;
    }

    public int getVest() {
        return vest;
    }

    public void setVest(int vest) {
        this.vest = vest;
    }

    @Override
    public String toString() {
        return "Boat {" +
                "peopleCount = " + peopleCount +
                ", vesla = " + vesla +
                ", chairs = " + chairs +
                ", vest = " + vest + " " +
                super.toString() +
                "}\n";
    }

}

File: lab 11(lab 2 python)/boat.py
from equipment import Equipment


class Boat(Equipment):

    def __init__(self,produce_name, weight, people_count,vesla,chairs,vest):
        super(Boat, self).__init__(produce_name, weight)
        self.people_count = people_count
        self.vesla = vesla
        self.chairs = chairs
        self.vest = vest

    def __str__(self):
        return super(Boat, self).__str__() + 'people_count = ' + str(self.people_count) \
               + 'vesla = ' + str(self.vesla) + 'chairs = ' + str(self.chairs) + 'vest = ' + str(self.vest)